saga/order-service/src/main/resources/META-INF/resources/index.html

```diff
@@ -0,0 +1,256 @@
+<!DOCTYPE html>
```

Comment:
same as above w.r.t. to auto-generated default index page

Reply:
Done.